indsl/equipment/operational_availability.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+
+
+def calculate_operational_availability(up_time_data: pd.Series, down_time_data: pd.Series) -> pd.Series:
+    r"""Calculate operational availability.
```

Comment:
Thats the name that will be shown in Charts. I suppose we dont need `Calculate`?